channels/vn.m3u

```diff
@@ -102,6 +102,8 @@ http://113.161.4.48:8080/phuyen/tv.m3u8
 http://113.161.6.157:8081/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8
 #EXTINF:-1 tvg-id="QuocHoi.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://www.quochoitv.vn/images/logo-qhtv.png" group-title="",Quốc Hội (720p)
 http://113.164.225.140:1935/live/quochoitvlive.stream_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="SkyShop.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",SkyShop
+https://livecdn.fptplay.net/sda/skymart_hls.smil/playlist.m3u8
```

Comment:
```suggestion
#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
https://livecdn.fptplay.net/sda/skymart_hls.smil/playlist.m3u8
```

Time to move them into a separate file

Reply:
Yes, let me see what I can manage to get.

https://fptplay.vn/